lms/static/sass/discussion/views/_thread.scss

```diff
@@ -134,6 +134,10 @@ body.discussion {
         border-radius: 3px;
       }
     }
+
+    img {
```

Comment:
Note that this might not apply to all three discussion locations (the discussion board, the inline discussion, and the teams discussion) because I'm not sure they all have ``discussion`` on their body tags. As part of my work on the discussion header, I'm removing the ``body.discussion`` selector so that they can apply wherever discussions are shown:

https://github.com/edx/edx-platform/pull/12678